gcovr/utils.py

```diff
@@ -70,7 +70,7 @@ def realpath(path):
 
 
 def get_os_independent_path(path):
-    return path.replace(os.path.sep, SEPARATOR)
+    return path.replace(os.path.sep, GCOVR_PATH_SEPARATOR)
```

Comment:
This function does the same than `force_unix_separator` but is only used here. Can you remove this function and replace the usage with `def sort_coverage`?

Reply:
This is no longer part of the diff.